### DISCUSSION TITLE
Project Reunion First Servicing Release 0.5.5 is Now Available!!

### DISCUSSION THREAD

**Discussion by dayman40** [microsoft/WindowsAppSDK#735] [Announcements]
Project Reunion 0.5.5 is a stable servicing release that fixes a few critical bugs. 
 
If you're building an app with Project Reunion for the first time, follow the [installation instructions here](https://docs.microsoft.com/windows/apps/project-reunion/get-started-with-project-reunion).
 
If you have an app that's already using Project Reunion 0.5, you can update to the latest release by [following the instructions here](https://docs.microsoft.com/windows/apps/project-reunion/update-existing-projects-to-the-latest-release). 
 
 
### Bugs fixed in Project Reunion 0.5.5:
- Memory leaks caused by subscribing to some framework element events and page navigation due to a C#/WinRT issue
 
- Alt+F4 does not close desktop apps
 
- Drag and drop does not work with touch or pen due to an issue with StartDragAsync()
 
- Black flash on WebView2 when setting Visibility to Collapsed
 
- Microsoft.ProjectReunion.Foundation is breaking WPF compilations
 
- Installing the Project Reunion 0.5 NuGet package results in build failures in projects that target AnyCPU
 
- Crash when panel containing SwapChainPanel is opened multiple times
 
- Acrylic brush renders transparent
 
- A few bugs having to do with GC/ObjectDisposedExceptions, marshaling value and nullable types (TimeSpan, IReference<Vector3>, etc) in C#/WinRT are now fixed. **In order to get these GC fixes, you must use the latest .NET 5.0.6 SDK** which will be available in May. See our [update instructions](https://docs.microsoft.com/en-us/windows/apps/project-reunion/update-existing-projects-to-the-latest-release) for how you can explicitly reference the latest 5.0.6 SDK.

**Comment by mrlacey** [comment]
I think there's a typo, **0.5.5** has just been released.
As per [this announcement on Twitter](https://twitter.com/WindowsUI/status/1383111182110711809), the [above-linked upgrade doc](https://docs.microsoft.com/en-us/windows/apps/project-reunion/update-existing-projects-to-the-latest-release), and the [VSIX marketplace version](https://marketplace.visualstudio.com/items?itemName=ProjectReunion.MicrosoftProjectReunion). 
But not yet shown as a [release](https://github.com/microsoft/ProjectReunion/releases)

**Reply by mrlacey** [reply]
well edited 😉

**Reply by andrewleader** [reply]
Haha 😊 the version number was going to be 0.5.4 yesterday 😜 Thanks for pointing it out!

**Reply by dayman40** [reply]
Thank you so much for pointing this out @mrlacey! :)
